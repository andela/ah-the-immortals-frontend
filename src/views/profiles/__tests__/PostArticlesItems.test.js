import { mount } from 'enzyme';
import configureMockStore from 'redux-mock-store';
import MutationObserver from 'mutation-observer';
import { Provider } from 'react-redux';
import thunk from 'redux-thunk';
import React from 'react';
import { MemoryRouter } from 'react-router-dom';
import ArticleItem from '../ProfileArticlesItems';

global.MutationObserver = MutationObserver;
document.getSelection = jest.fn();

describe('test getting articles for a profile', () => {
  let testStore = configureMockStore([thunk]);
  let store = testStore({ profile: {} });
  let props = {
    article: {}
  };
  const wrapper = mount(
    <MemoryRouter>
      <Provider store={store}>
        <ArticleItem {...props} />
      </Provider>
    </MemoryRouter>
  );
  it('Match profile articles', () => {
    expect(wrapper).toMatchSnapshot();
  });

});
