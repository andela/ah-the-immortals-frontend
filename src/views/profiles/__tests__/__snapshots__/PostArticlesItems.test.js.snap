// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test getting articles for a profile Match profile articles 1`] = `ReactWrapper {}`;
